### ISSUE TITLE
Telegram API's json response differs between a DM and a watched channel

### ISSUE THREAD

**Issue by xmcqueen** [antirez/stonky#4]
For a direct message to the bot, the `message` in the json payload is in a field called "message", but for a message posted to a watched channel, the `message` in the json payload is in a field called "channel_post".  See the `jq` expressions below.

```
#chan_post.json is the json payload returned when a channel post is detected 
(36) bmcqueen-mn2:stonky bmcqueen$ cat chan_post.json  | jq .result[0].channel_post.text
"$dldld"

#dm_post.json is the json payload returned when a direct message to the bot is detected
(36) bmcqueen-mn2:stonky bmcqueen$ cat dm_post.json | jq .result[0].message.text
"$helpq"
(36) bmcqueen-mn2:stonky bmcqueen$ 

```

PR incoming

**Comment by xmcqueen** [comment]
I just noticed it yesterday, when I started messing with stonky.  Maybe they are rolling out an api change.  I'm in sf bay area.  Here's some data showing the responses when calling the API.  Note the message from the general channel are in `.result[0].channel_post`, but the message to the bot directly the messages is at
`.result[0].message`.

2. I setup a channel names *tchan*
3. and a -botfather- bot named *qbot* and qbot is an *admin* in the tchan channel
4. i go to my phone and use the telegram app
5. then send 2 messages
6. first a message to qbot directly via the regular user/bot *qbot*
7. secondly a message to the tchan channel which qbot should see and pickup
8. next I pull the json of pending messages from the API
9. and check the messages. They are all there, but not all in `.result[].messages`

```
#call the telegram API and show there's nothing waiting
(36) bmcqueen-mn2:stonky bmcqueen$ curl -s 'https://api.telegram.org/botxxxx/getUpdates?offset=-99&timeout=1&allowed_updates=message' | jq .
{
  "ok": true,
  "result": []
}


# send the two test messages
# call the telegram API again
# see the messages shown below
# the direct message to qbot is in .result[0].message
# the channel message is at .result[1].channel_post
# note the second message is NOT at ".message"
(36) bmcqueen-mn2:stonky bmcqueen$ curl -s 'https://api.telegram.org/botxxxx/getUpdates?offset=-99&timeout=1&allowed_updates=message' | jq .
{
  "ok": true,
  "result": [
    {
      "update_id": 517926475,
      "message": {
        "message_id": 38,
        "from": {
          "id": 1530095388,
          "is_bot": false,
          "first_name": "Self",
          "last_name": "Mcswayne",
          "language_code": "en"
        },
        "chat": {
          "id": 1530095388,
          "first_name": "Self",
          "last_name": "Mcswayne",
          "type": "private"
        },
        "date": 1614194721,
        "text": "$$ msg direct to qbot"
      }
    },
    {
      "update_id": 517926476,
      "channel_post": {
        "message_id": 75,
        "sender_chat": {
          "id": -1001233660747,
          "title": "Tchan",
          "username": "teachange",
          "type": "channel"
        },
        "chat": {
          "id": -1001233660747,
          "title": "Tchan",
          "username": "teachange",
          "type": "channel"
        },
        "date": 1614194763,
        "text": "$$ message in tchan and qbot should see it and pick it up"
      }
    }
  ]
}
(36) bmcqueen-mn2:stonky bmcqueen$ 
```

**Comment by xmcqueen** [comment]
fixed by @antirez [f2cc9bd43ebcea2c3111046a8a4672413898bc9f](https://github.com/antirez/stonky/commit/f2cc9bd43ebcea2c3111046a8a4672413898bc9f)
